CMakeLists.txt

```diff
@@ -463,9 +474,22 @@ if (APPLE)
         endif ()
 
         if (NOT BUILD_PYTHON)
+            if ("${CMAKE_OSX_ARCHITECTURES}" STREQUAL "arm64")
+                file (GLOB INSTALL_GUI_FRAMEWORKS ${PYTHONDIR})
+                set (PYTHON_DESTINATION "${INSTALL_LIB_DIR}/../Resources")
+            else()
+                file (GLOB INSTALL_GUI_FRAMEWORKS ${PYTHONPATH})
+                set (PYTHON_DESTINATION "${INSTALL_LIB_DIR}")
+            endif()
 		    install (
-		    	DIRECTORY ${INSTALL_GUI_FRAMEWORKS}
-		    	DESTINATION ${INSTALL_LIB_DIR}
```

Comment:
Installed libraries should go in `${INSTALL_LIB_DIR}`

Reply:
I'm not sure what your implying.  The destination is indeed ${INSTALL_LIB_DIR}.

Reply:
This line is deleted and replaced with something else.

Reply:
I see what you mean.  This is for the same reason in one of our previous comments, which is that Python's modules cannot be signed in the /Frameworks directory.

Alternatively, I believe we could put the libpython in /Frameworks and set PYTHONHOME to the module path.  I'm not sure if keeping all libraries in /Frameworks is worth the extra complexity and environment modification though.